addons/website/static/src/scss/website.wysiwyg.scss

```diff
@@ -10,13 +10,6 @@
     // Override css variables to influence the default style of the editor
     // without duplicating the css.
     @include print-variable('o-we-toolbar-height', $o-navbar-height);
-
-    // Need info about the color of each color palette
-    @for $index from 1 through length($o-color-palettes) - 1 { // Not the user one
-        @each $name, $color in nth($o-color-palettes, $index) {
-            @include print-variable('o-palette-#{$index}-#{$name}', $color);
-        }
-    }
```

Comment:
I put it here because it made it available for the web.frontend_layout I use for my survey. But ok I can move it. It will be more chipotage for the assets

Reply:
Yes but the more reason we need this chipotage ;) We don't want to flood the visitors CSS because of the configuration starting screen ;)